oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/AuthorizationCodeLiveTest.java

```diff
@@ -17,63 +17,72 @@
 
 public class AuthorizationCodeLiveTest {
     public final static String AUTH_SERVER = "http://localhost:8083/auth/realms/baeldung/protocol/openid-connect";
-    public final static String RESOURCE_SERVER = "http://localhost:8081/resource-server"; 
+    public final static String RESOURCE_SERVER = "http://localhost:8081/resource-server";
     private final static String REDIRECT_URL = "http://localhost:8082/new-client/login/oauth2/code/custom";
-	private final static String CLIENT_ID = "newClient";
-	private final static String CLIENT_SECRET = "newClientSecret";
+    private final static String CLIENT_ID = "newClient";
+    private final static String CLIENT_SECRET = "newClientSecret";
 
     @Test
     public void givenUser_whenUseFooClient_thenOkForFooResourceOnly() {
         final String accessToken = obtainAccessTokenWithAuthorizationCode("john@test.com", "123");
 
-        final Response fooResponse = RestAssured.given().header("Authorization", "Bearer " + accessToken).get(RESOURCE_SERVER + "/api/foos/1");
+        final Response fooResponse = RestAssured.given()
+            .header("Authorization", "Bearer " + accessToken)
+            .get(RESOURCE_SERVER + "/api/foos/1");
         assertEquals(200, fooResponse.getStatusCode());
-        assertNotNull(fooResponse.jsonPath().get("name"));
+        assertNotNull(fooResponse.jsonPath()
+            .get("name"));
 
     }
 
     private String obtainAccessTokenWithAuthorizationCode(String username, String password) {
-    	
-
-		String authorizeUrl = AUTH_SERVER + "/auth";
-		String tokenUrl = AUTH_SERVER + "/token";
-
-		Map<String, String> loginParams = new HashMap<String, String>();
-		loginParams.put("client_id", CLIENT_ID);
-		loginParams.put("response_type", "code");
-		loginParams.put("redirect_uri", REDIRECT_URL);
-		loginParams.put("scope", "read write");
-
-		// user login
-		Response response = RestAssured.given().formParams(loginParams).get(authorizeUrl);
```

Comment:
let's revert the changes apart from the openid one if we can please?

Reply:
changes done